app/dashboard/templates/bulk_payout_bounty.html

```diff
@@ -20,6 +20,33 @@
   <head>
     {% include 'shared/head.html' %}
     {% include 'shared/cards.html' %}
+    <link rel="stylesheet" type="text/css" href="{% static "v2/css/tooltip_hover.css" %}">
+    <style type="text/css">
```

Comment:
is there a reason this css is inline instead of in `tooltip_hover.css`?

Reply:
It is basically reusing the `#tooltip--hover` design primarily designed keeping `process_bounty.html` in mind, and making a few changes to it.